src/cmd/report.go

```diff
@@ -19,42 +19,43 @@ var (
 	dhFlag   string
 	oFlag    string
 
-	// fetchCmd represents the fetch command
-	fetchCmd = &cobra.Command{
-		Use:   "fetch",
-		Short: "Fetch install data from Brew, GitHub, and/or Docker Hub",
+	// reportCmd represents the report command
+	reportCmd = &cobra.Command{
+		Use:     "report",
+		Aliases: []string{"fetch"},
+		Short:   "report install data from Brew, GitHub, and/or Docker Hub",
 		Run: func(cmd *cobra.Command, args []string) {
 
 			if brewFlag != "" {
-				fetchBrewData(brewFlag)
+				reportBrewData(brewFlag)
 				fmt.Println("")
 			}
 
 			if ghFlag != "" {
-				fetchGitHubData(ghFlag)
+				reportGitHubData(ghFlag)
```

Comment:
These functions should still be fetch as they is what they are doing. The command is reporting, not these functions.